docs/api-updates.md

```diff
@@ -14,7 +14,7 @@ create a pull request to update Cloudflare docs at the same time.
 
 ## Compatibility documentation in Cloudflare docs
 
-Compatibility flags and dates should be added to the [Cloudflare Workers Compatibility Dates](https://developers.cloudflare.com/workers/platform/compatibility-dates) documentation.
+Compatibility flags and dates should be added to the [Cloudflare Workers Compatibility flags](https://developers.cloudflare.com/workers/configuration/compatibility-flags/) and [dates](https://developers.cloudflare.com/workers/configuration/compatibility-dates/) documentation.
```

Comment:
Thank you for filing this PR! I think we only need the first link here – newly added compatibility flags generally have a specified compatibility date from which they start being active, but this does not require updating the compatibility dates docs page per se.